styles.css

```diff
@@ -1,267 +1,119 @@
 
-/* TYPOGRAPHY */
-header                                              {font-family: "expressway", sans-serif; font-style: normal}
-#slivers h2, #slideshow                             {font-family: Georgia}
-#slivers h2                                         {font-size: 22px; line-height: 24px; font-weight: normal}
-.navbar-nav, footer                                 {font-size: 11px}
-.navbar-nav a                                       {font-weight: bold}
-footer a:focus, footer a:hover                      {text-decoration: none}
-#slideshow blockquote                               {font-size: 16px}
-#slideshow p.more                                   {font-size: 14px; font-style: italic}
+/* BACKGROUNDS */
+body                                                {background-image: url(images/topbluefade.gif); background-repeat: repeat-x}
+.slivers                                            {background-image: url(images/herobackground.gif)}
+@media (min-width: 768px) {header.container         {background-image: url(images/rails.png); background-repeat: no-repeat; background-position: 22px 0px}}
 
-footer                                              {text-align: center}
-#announce                                           {font-size: 11px; line-height: 13px}
-#slivers p                                          {font-size: 10px}
-h1                                                  {font-size: 38px; line-height: 30px; font-weight: 700}
-header h2                                           {font-size: 21px; line-height: 24px; font-weight: 400}
-@media (max-width: 768px) {
-  h1                                                {font-size: 31px}
-  header h2                                         {font-size: 18px}
-}
+/* COLORS */
+a                                                   {color: #369}
+a:hover, a:focus                                    {background-color: #f4dda6}
+mark                                                {background-color: #ffc}
+footer                                              {color: #666}
+#menu a, code, pre, footer a                        {color: #000}
+#menu a:focus, #menu a:hover                        {color: #fff; background-color: #333}
+.navbar-default, code, pre                          {background-color: transparent}
+.navbar                                             {border-color: transparent}
+h4                                                  {color: #999}
+.framed img, .col-sm-4 img                          {background-color: #fff; border-color: #ccc}
+h5                                                  {color: #900}
+cite                                                {color: #a69e8a}
+#announce                                           {background-color: #981a21}
+#announce a, .inverted a:hover, .inverted a:focus   {color: #fff}
+.slivers                                            {border-top-color: #f6e69f; border-bottom-color: #f6e69f}
```

Comment:
Oh, thank you for this great explanation. I see now the benefits of this style. On the beginning I thought you were mixing different types of properties on the same declaration, so colors and positions properties would be mixed in the same line, what is not good for maintainability. After this explanation I see that you were opening multiple declarations for each type of properties and things are grouped what makes sence.